Readability.js

```diff
@@ -896,6 +896,14 @@ Readability.prototype = {
       let shouldRemoveTitleHeader = true;
 
       while (node) {
+
+        if(node.tagName === "HTML") {
```

Comment:
This should have a whitespace between the `if` and opening  bracket. This is also causing the tests to fail: https://community-tc.services.mozilla.com/tasks/groups/Lqf0BNjvQoSSmbvA9CS_AA 

Once that's addressed you should run `npm run lint && npm test` to make sure we don't have any other linting/test failures. See contributing guide [here](https://github.com/mozilla/readability/blob/master/CONTRIBUTING.md#tests) for more info.